code/modules/ai/presets/xeno/sentinel.dm

```diff
@@ -0,0 +1,10 @@
+/mob/living/carbon/xenomorph/sentinel/ai
+
+/mob/living/carbon/xenomorph/sentinel/ai/Initialize()
+	. = ..()
+	AddComponent(/datum/component/ai_behavior, /datum/ai_mind/carbon/xeno/sentinel)
+	a_intent = INTENT_HARM
+	//Cheat codes
+	xeno_caste.caste_flags += CASTE_INNATE_HEALING
```

Comment:
this will affect ALL sentinels

Reply:
same for the other changes to xeno_caste

Reply:
Wasn't able to replicate by first spawning a ai sentinel then spawning a non ai sentinel, taking over it and using the ability until about 500 plasma; plasma only regenerated at 1 per tick

Reply:
Retract my statement, cheat codes will be removed:tm: